api/mod-record-specifications.yaml

```diff
@@ -55,4 +54,9 @@ paths:
   /specification-storage/indicators/{indicatorId}/indicator-codes:
     $ref: 'paths/specification-storage/indicator-codes-path.yaml'
   /specification-storage/indicator-codes/{indicatorCodeId}:
-    $ref: 'paths/specification-storage/indicator-codes-by-id-path.yaml'
\ No newline at end of file
+    $ref: 'paths/specification-storage/indicator-codes-by-id-path.yaml'
+
+components:
+  schemas:
+    specificationUpdatedEvent:
```

Comment:
You should define the event as a usual class since it's not part of REST API. It will not be added to any documentation, so it doesn't make sense.

Reply:
But it'll be available in dependant modules this way for consuming

Reply:
You can just create it in dto submodule